modules/deploy/pages/setting-up-dr-cluster.adoc

```diff
@@ -40,33 +41,67 @@ It serves an important role in supporting Disaster Recovery (DR) and Data Migrat
 
 == Recommended Deployment Models
 
+
+=== Zero Downtime Active-Active Disaster Recovery
+// tag::zero-downtime-active-active[]
+
+This model provides true zero-downtime disaster recovery using bi-directional XDCR between two active mobile clusters.
+Both clusters remain operational, with no fail-over through load balancer switching.
+You must configure both clusters with `shared_bucket_access=true` and `import_docs=true`.
+
+
+.Set Up
+To set up zero-downtime disaster recovery:
+
+. Configure bi-directional XDCR between the Primary and disaster recovery clusters. +
+Enable automatic filtering of cluster specific metadata.
+. Deploy Sync Gateway in active mode on both clusters.
+. Configure load balancer to route traffic primarily to the Primary cluster.
+. Verify replication health between the two active clusters.
+
+[#fig-dr-active-active-setup]
+.DR Cluster Setup (Active-Active)
+image::ROOT:{image-xdcr-active-active-replication}[,{std-image-size}]
+
+
+.Activation
+To activate disaster recovery:
+
+. Update load balancer configuration to redirect traffic to the disaster recovery cluster. +
+This process requires no Sync Gateway service interruption.
+. Verify disaster recovery cluster is handling traffic properly.
+. Maintain bi-directional replication for recovery preparedness. +
+The original primary becomes new DR cluster automatically and no manual XDCR reconfiguration required
+
+// end::zero-downtime-active-active[]
+
 === Clusters in Same Region
 // tag::clusters-in-same-region[]
 
-This model caters for situations where the Active and Disaster Recovery clusters are in the same region or data center -- see: <<fig-dr-same-regn>>.
-It includes an optional optimization step, which will ensure that there is no downtime during the activation stage.
+This model caters for situations where the Active and Disaster Recovery clusters are in the same region or datacenter -- see: <<fig-dr-same-regn>>.
+It includes an optional optimization step, which verifies that there is no downtime during the activation stage.
 
 .Set Up
 To set up and maintain a disaster recovery cluster:
 
-. [_Optional step -- for optimization_] Connect Sync Gateway to the Disaster Recovery cluster just long enough to create indexes.
+. [*Optional step -- for optimization*] Connect Sync Gateway to the Disaster Recovery cluster just long enough to create indexes.
```

Comment:
I would change the recommendation here to start the database in offline mode on the DR cluster.